rllib/policy/dynamic_tf_policy.py

```diff
@@ -650,7 +650,7 @@ def _get_input_dict_and_dummy_batch(self, view_requirements, existing_inputs):
         input_dict = {}
         for view_col, view_req in view_requirements.items():
             # Point state_in to the already existing self._state_inputs.
-            mo = re.match("state_in_(\d+)", view_col)
+            mo = re.match(r"state_in_(\d+)", view_col)
```

Comment:
This means this didn't work before, right?

Reply:
I'm pretty sure it _did_ work before, but only because escape sequences that aren't recognized by python are kept as literal characters. [From the python docs:](https://docs.python.org/3/reference/lexical_analysis.html#string-and-bytes-literals)

> Unlike Standard C, all unrecognized escape sequences are left in the string unchanged, i.e., the backslash is left in the result.

So these should have been working just fine, but in some cases there will be unintended behavior - for example, escaping a newline with a backslash in a string will cause the newline to be ignored:

```python
>>> 'This string will not include \
... backslashes or newline characters.'
'This string will not include backslashes or newline characters.'
```

There are two places where I found bad escape sequences:

1. Docstrings with LaTeX markup
2. Regexes

In both cases, you want the string to literally contain the characters inside, so it makes sense to make them string literals here. Eventually, unrecognized escape sequences will generate a `SyntaxError`, so these need to get fixed anyway.